schema/schema_test.go

```diff
@@ -0,0 +1,175 @@
+package schema_test
+
+import (
+	"bufio"
+	"bytes"
+	"io/ioutil"
+	"testing"
+
+	"github.com/hpcloud/termui/schema"
+
+	term "github.com/cloudfoundry/cli/cf/terminal"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+// TestSchema tests schema package
+func TestSchema(t *testing.T) {
+	RegisterFailHandler(Fail)
+
+	RunSpecs(t, "Schema")
+}
+
+var _ = Describe("Schema", func() {
+	Describe("Test if schema asks correctly for a string", func() {
+		Context("ask for string", func() {
+			It("should be Insert string value for /stringtest [required]", func() {
+
+				str := `{
+   "type":"object",
+   "properties":{
+      "stringtest":{
+         "type":"string"
+      }
+   },
+   "required":[
+      "stringtest"
+   ]
+}`
+
+				in := bytes.Buffer{}
+				out := bytes.Buffer{}
+				writer := bufio.NewWriter(&out)
+
+				ui := term.NewUI(
+					&in,
```

Comment:
Since `in` is only used here, we can pass `&bytes.Buffer{}` directly (so when reading this we don't have to think about whether we use it elsewhere).  Not sure if that's actually better style, though - the name can be useful too. Please use your judgment?